src/Ledger/Certs.lagda

```diff
@@ -426,20 +426,25 @@ data _⊢_⇀⦇_,CERT⦈_ where
 data _⊢_⇀⦇_,CERTBASE⦈_ where
 \end{code}
 \begin{code}
-  CERT-base :
-    let open PParams pp
-        refresh         = mapPartial getDRepVote (fromList vs)
-        refreshedDReps  = mapValueRestricted (const (e + drepActivity)) dreps refresh
-        wdrlCreds       = mapˢ stake (dom wdrls)
+  CERT-base : let
+    open PParams pp
+    refresh          = mapPartial getDRepVote (fromList vs)
+    refreshedDReps   = mapValueRestricted (const (e + drepActivity)) dReps refresh
+    wdrlCreds        = mapˢ stake (dom wdrls)
+    freshVoteDelegs  = voteDelegs ∣^ mapˢ (credVoter DRep) (dom refreshedDReps)
```

Comment:
If it fits on the page, I think it's better to inline this. You can also just use `dReps` instead of `refreshedDReps`, since they have the same domain

Reply:
Yes, the domain is the same, but do you think using `refreshedDReps` makes the point of the restriction to existing DReps clearer?  Well, I guess not, since the refresh doesn't change the set of existing DReps.  Okay, I'll change it back to `dReps`.

Reply:
Inlining makes the code run far outside the figure box, so I'll change back to an extra variable definition, `validVoteDelegs`, for the restricted `voteDelegs` (restricted to registered DReps).